src/lib/ui/containers/SignupForm/SignupForm.svelte

```diff
@@ -0,0 +1,158 @@
+<script lang="ts">
+  import { createEventDispatcher, getContext } from "svelte";
+  import {
+    Errors,
+    Field,
+    FormError,
+    Select,
+    faivform,
+  } from "@tuentyfaiv/svelte-form";
+  import { getTexts } from "$lib/logic/utils/objects.js";
+  import { fieldsSignup } from "$lib/logic/schemas/index.js";
+
+  import type { Readable } from "svelte/store";
+  import type { FieldInputForm } from "$lib/logic/typing/globals/interfaces.js";
+  import type { SignupValues } from "$lib/logic/typing/schemas/auth.js";
+  import type { ConfigForm } from "$lib/logic/typing/store/config.js";
+  import type { Props, SignupFields } from "./SignupForm.proptypes.js";
+
+  import * as stylesinternal from "./SignupForm.styles.js";
+
+  export let submit: Props["submit"];
+  export let context: Props["context"] = "form";
+  export let showErrors: Props["showErrors"] = true;
+  export let options: Props["options"] = [];
+  export let code: Props["code"] = "bycountry";
+  export let confirm: Props["confirm"] = true;
+  export let styles: Props["styles"] = undefined;
```

Comment:
🚫 **[eslint]** <[no-undef-init](https://eslint.org/docs/latest/rules/no-undef-init)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>It's not necessary to initialize 'styles: Props["styles"]' to undefined.
```suggestion
  export let styles: Props["styles"];
```